fastestimator/estimator.py

```diff
@@ -115,26 +104,26 @@ def _prepare_traces(self):
         traces.insert(0, TrainEssential(monitor_names=monitor_names))
         modes = self.pipeline.get_modes() - {"test"}
         if "eval" in modes:
-            traces.append(EvalEssential(loss_keys=loss_keys))
+            traces.insert(1, EvalEssential(loss_keys=loss_keys))
```

Comment:
why insert instead of append?

Reply:
eval essential before all other user provided traces in case those traces want to look at the eval essential output